docroot/sites/settings/common.settings.php

```diff
@@ -43,8 +43,8 @@
   'secure' => FALSE,
   'httponly' => FALSE,
   'role' => [
-    'population' => 'administrator:eduPersonEntitlement,=,hsdo:web|administrator:eduPersonEntitlement,=,itservices:webservices',
-    'eval_every_time' => TRUE,
+    //    'population' => 'administrator:eduPersonEntitlement,=,hsdo:web|administrator:eduPersonEntitlement,=,itservices:webservices',
```

Comment:
Why is this going away?

Reply:
To allow site configuration of workgroups. for example all math workgroups will get the role site manager

Reply:
I might have to adjust config ignore or something later